nhais/mesh/mesh_outbound.py

```diff
@@ -1,17 +1,22 @@
 import utilities.integration_adaptors_logger as log
-from comms import queue_adaptor
+from comms import proton_queue_adaptor
 from retry import retriable_action
 from utilities import timing, config
 
 logger = log.IntegrationAdaptorsLogger(__name__)
 
 class MeshOutboundWrapper:
 
-    def __init__(self,
-                 queue_adaptor: queue_adaptor.QueueAdaptor = None):
-        self.queue_adaptor = queue_adaptor
-        self.queue_max_retries = int(config.get_config('MAX_RETRIES', default='10'))
-        self.queue_retry_delay = float(config.get_config('RETRY_DELAY', default='0.1'))
+    def __init__(self):
+        self.queue_adaptor = proton_queue_adaptor.ProtonQueueAdaptor(
+            urls=config.get_config('OUTBOUND_QUEUE_BROKERS').split(','),
+            queue=config.get_config('OUTBOUND_QUEUE_NAME'),
+            username=config.get_config('OUTBOUND_QUEUE_USERNAME', default=None),
+            password=config.get_config('OUTBOUND_QUEUE_PASSWORD', default=None),
+            max_retries=int(config.get_config('OUTBOUND_QUEUE_MAX_RETRIES', default='3')),
+            retry_delay=int(config.get_config('OUTBOUND_QUEUE_RETRY_DELAY', default='100')) / 1000)
+        self.queue_max_retries = int(config.get_config('OUTBOUND_MAX_RETRIES', default='3'))
```

Comment:
Why those 2 on lines 18 and 19 are here? Line 16 and 17 already sets those values